app/src/main/res/layout/activity_orgs.xml

```diff
@@ -66,8 +64,22 @@
                 android:fontFamily="sans-serif-light"
                 android:text="Association for Computing Machinery"
                 android:textColor="@color/white"
-                android:textStyle="bold"
-                android:textSize="20sp" />
+                android:textSize="20sp"
+                android:textStyle="bold" />
+
+            <TextView
+                android:id="@+id/tv_address"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_below="@+id/tv_name"
+                android:layout_centerHorizontal="true"
+                android:layout_marginTop="15sp"
+                android:fontFamily="sans-serif-light"
+                android:text="Description"
+                android:textAllCaps="true"
+                android:textColor="@color/white"
+                android:textSize="20sp"
+                android:textStyle="bold" />
```

Comment:
Might want to refactor the id to tv_description, but other than that, LGTM, I'll implement with this immediately

Reply:
lgtm. can't see any conflicts.

Reply:
Looks good, I didn't get any errors.